README.md

```diff
@@ -141,18 +141,25 @@ yarn install
 to do that.
 
 ## Changing the database model
+
 When the database model in models.py is changed a new migration has to be created specifying how to change to the new
 database format. This can be done by
-```
+
+```shell
 python manage.py makemigrations --settings=config.settings.local
 ```
 
 Afterwards the test database has to be updated as well. Use the dumpdata command to generate a test database from the
 currently running database:
-```
+
+```shell
 python manage.py dumpdata --settings=config.settings.local > e2e_tests/database/test_database.json
 ```
 
+On Windows (and possibly others): Check the diff of `e2e_tests/database/test_database.json`.
+Possibly use a formatter on the generated file.
+Check for special characters, e.g. German umlauts.
```

Comment:
I agree that this is just a rough hint, but I would have thought that such a hint is better than nothing. When actually running into the problem, one would probably know, what to do.

When I did the dump I got a single line and had to use the formatter to get a file somewhat similar to the one in the repo. In the diff there still were a bunch of changes unrelated to the model change, which I simply reverted.

Since I am not sure, if this works all the time, I prefer to just give some hints and not a longish brain-switched-off recipe.

Reply:
I see, so it's pretty complicated.
Maybe we can mention this with something like:
"Check the diff of ... for any unexpected parts and adjust as necessary."

Reply:
Will change.